packages/dds/tree/src/test/simple-tree/treeApi.spec.ts

```diff
@@ -768,5 +768,147 @@ describe("treeNodeApi", () => {
 			assert.equal(nodeChanged, true, "'nodeChanged' should have fired");
 			assert.equal(treeChanged, true, "'treeChanged' should have fired");
 		});
+
+		it(`'nodeChanged' includes the names of changed properties (objectNode)`, () => {
+			const sb = new SchemaFactory("test");
+			class treeSchema extends sb.object("root", {
+				prop1: sb.optional(sb.number),
+				prop2: sb.optional(sb.number),
+				prop3: sb.optional(sb.number),
+			}) {}
+
+			const view = getView(new TreeViewConfiguration({ schema: treeSchema }));
+			view.initialize({ prop1: 1, prop2: 2 /* , prop3: 0 */ });
```

Comment:
I don't think this inline comment is helpful. The default for it isn't 0, its undefined, so its misleading. If you want to be explicit you can do:
```suggestion
			view.initialize({ prop1: 1, prop2: 2, prop3: undefined });
```
but I think this would be fined:

```suggestion
			view.initialize({ prop1: 1, prop2: 2 });
```

Reply:
Yeah, I'll remove it, it's a leftover from when I was testing this and getting errors applying the change on the branch through the wrong editor APIs.